javascript.md

```diff
@@ -605,7 +605,7 @@ ruleOfThree(4, 2, 6);
 
 ## Polyfills
 
-Where possible use the native browser implementation and include [a polyfill that provides that behavior][27] for unsupported browsers. This makes the code easier to work with and less involved in hackery to make things just work.
+Where possible use the native browser implementation and include [a polyfill that provides that behavior][27] for unsupported browsers. This makes the code easier to work with and less involved in hackery to make things just work, the exception being Internet Explorer 8. Although we continue to support IE8, we do not support scripted interactions and rely instead on progressive enhancement to provide IE8 users with a functioning experience.
```

Comment:
"the exception being Internet Explorer 8" does not logically follow from the first clause of that sentence. I would probably break out our position on IE8 into a totally new paragraph.